addons/l10n_om/data/template/account.tax-om.csv

```diff
@@ -0,0 +1,41 @@
+"id","sequence","name","description","invoice_label","price_include","amount","amount_type","type_tax_use","tax_group_id","repartition_line_ids/factor_percent","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id","description@ar"
+"l10n_om_sale_vat_5","1","5%",,"5%","False","5","percent","sale","tax_group_vat_5",,"base","invoice","+1a_B",,
+,,,,,,,,,,,"tax","invoice","+1a_T",,
```

Comment:
i know it will be a bit boring to do but could you add the "" where it need to be, meaning every empty element :smile: